app/finda-face/faceDetector.js

```diff
@@ -0,0 +1,14 @@
+var Faced = require('faced');
```

Comment:
took what you had in face checker and slimmed it down to basically return yes/no.  How long is this taking? hopefully with lower res image it will be fast